stl/inc/chrono

```diff
@@ -5918,6 +6000,112 @@ private:
     _Chrono_formatter<_CharT, false> _Impl;
 };
 
+template <class _Duration, class _CharT>
+struct formatter<_CHRONO sys_time<_Duration>, _CharT> {
+    formatter() {
+        _Impl._Time_zone_abbreviation = _STATICALLY_WIDEN(_CharT, "UTC");
+    }
+
+    auto parse(basic_format_parse_context<_CharT>& _Parse_ctx) {
+        return _Impl.template _Parse<_CHRONO sys_time<_Duration>>(_Parse_ctx);
+    }
+
+    template <class _FormatContext>
+    auto format(const _CHRONO sys_time<_Duration>& _Val, _FormatContext& _FormatCtx) {
+        return _Impl._Write(_FormatCtx, _Val, _Fill_tm(_Val));
+    }
+
+private:
+    _Chrono_formatter<_CharT, false> _Impl;
+};
+
+template <class _Duration, class _CharT>
+struct formatter<_CHRONO utc_time<_Duration>, _CharT> {
+    formatter() {
+        _Impl._Time_zone_abbreviation = _STATICALLY_WIDEN(_CharT, "UTC");
+    }
+
+    auto parse(basic_format_parse_context<_CharT>& _Parse_ctx) {
+        return _Impl.template _Parse<_CHRONO utc_time<_Duration>>(_Parse_ctx);
+    }
+
+    template <class _FormatContext>
+    auto format(const _CHRONO utc_time<_Duration>& _Val, _FormatContext& _FormatCtx) {
+        const auto _Sys = _CHRONO utc_clock::to_sys(_Val);
+        return _Impl._Write(_FormatCtx, _Val, _Fill_tm(_Sys));
+    }
+
+private:
+    _Chrono_formatter<_CharT, false> _Impl;
+};
+
+template <class _Duration, class _CharT>
+struct formatter<_CHRONO tai_time<_Duration>, _CharT> {
+    formatter() {
+        _Impl._Time_zone_abbreviation = _STATICALLY_WIDEN(_CharT, "TAI");
+    }
+
+    auto parse(basic_format_parse_context<_CharT>& _Parse_ctx) {
+        return _Impl.template _Parse<_CHRONO tai_time<_Duration>>(_Parse_ctx);
+    }
+
+    template <class _FormatContext>
+    auto format(const _CHRONO tai_time<_Duration>& _Val, _FormatContext& _FormatCtx) {
+        const auto _Sys = _CHRONO sys_time<_Duration>{_Val.time_since_epoch()}
+                        - (_CHRONO sys_days{_CHRONO year{1970} / 1 / 1} - _CHRONO sys_days{_CHRONO year{1958} / 1 / 1});
+        return _Impl._Write(_FormatCtx, _Val, _Fill_tm(_Sys));
+    }
+
+private:
+    _Chrono_formatter<_CharT, false> _Impl;
+};
+
+template <class _Duration, class _CharT>
+struct formatter<_CHRONO gps_time<_Duration>, _CharT> {
+    formatter() {
+        _Impl._Time_zone_abbreviation = _STATICALLY_WIDEN(_CharT, "GPS");
+    }
+
+    auto parse(basic_format_parse_context<_CharT>& _Parse_ctx) {
+        return _Impl.template _Parse<_CHRONO gps_time<_Duration>>(_Parse_ctx);
+    }
+
+    template <class _FormatContext>
+    auto format(const _CHRONO gps_time<_Duration>& _Val, _FormatContext& _FormatCtx) {
+        const auto _Sys = _CHRONO sys_time<_Duration>{_Val.time_since_epoch()}
+                        + (_CHRONO sys_days{_CHRONO year{1980} / 1 / _CHRONO Sunday[1]}
+                            - _CHRONO sys_days{_CHRONO year{1970} / 1 / 1});

```

Comment:
Screams in math `x / 1 / 1`

Reply:
In case this is confusing, it's specifying a date (in year/month/day order), not dividing.

Reply:
Yes, but ... still

Reply:
I plan to refactor this a bit in a followup commit - not because I think there's anything wrong with `/ 1 / 1` (it's idiomatic for `<chrono>`, just like how `cats + dogs` is not commutative for `string`, or how `my_path / dir / dir` is valid for `path`), but because the Standard depicts `January` etc. and I think this can be boiled down to a `constexpr` constant for readability and debug codegen.